Source/Details/_ASDisplayView.mm

```diff
@@ -358,14 +359,79 @@ - (void)tintColorDidChange
   [node tintColorDidChange];
 }
 
-- (BOOL)canBecomeFirstResponder {
-  ASDisplayNode *node = _asyncdisplaykit_node; // Create strong reference to weak ivar.
-  return [node canBecomeFirstResponder];
+#pragma mark UIResponder Handling
+
+#define HANDLE_RESPONDER_METHOD(__sel) \
```

Comment:
Confusing to have both macros named the same - maybe call this one HANDLE_VIEW_RESPONDER_METHOD and the other one HANDLE_NODE_RESPONDER_METHOD?

Reply:
Renamed them.